### ISSUE TITLE
Notification inside Docker  -- SendMail

### ISSUE THREAD

**Issue by darkiron71** [hashtopolis/server#952]
Currently running Docker .14

We are attempting to use the notifications function inside the docker version. In the non-docker version the notification feature worked fine with send mail. What is the solution for the notifications to be able to send mail in the docker version. Does anything need to be configured differently?

**Comment by zyronix** [comment]
Good point, I guess we have to add some environment variables to configure the dockerfile.
